src/include/user/kpb.h

```diff
@@ -14,7 +14,7 @@
 struct sof_kpb_config {
 	uint32_t size; /**< kpb size in bytes */
 	uint32_t caps; /**< SOF_MEM_CAPS_ */
-	uint32_t no_channels; /**< no of channels */
+	uint32_t channels; /**< number of channels */
```

Comment:
This part will need a similar update on any tooling given it's user ABI ?

Reply:
Well, as you can see only variable names have changed and since we keep this initial values in the topology it doesn't matter. For our python tests however they mirrored the structure so to keep both of them in exactly the same shape I will ask @wwittbrx to update on their end.